### PR TITLE
Docs/rust target

### PR DESCRIPTION
**Pull request by ccamel**
Add links to Rust targets in `README.md` (+ minor fixes on links).

### CODE REVIEW

File: rust/cosmos_sdk_grpc_client/README.md [hunk 1]
```diff
@@ -3,7 +3,6 @@
 > Rust gRPC client for CØSMOS SDK based blockchains.
 
 [![build](https://github.com/okp4/okp4-cosmos-proto/actions/workflows/build.yml/badge.svg)](https://github.com/okp4/okp4-cosmos-proto/actions/workflows/build.yml)
-[![PyPI](https://img.shields.io/crates/v/cosmos_sdk_grpc_client)](https://crates.io/crates/cosmos_sdk_grpc_client)
```
Comment (warstrolo) [inline rust/cosmos_sdk_grpc_client/README.md:6]: Why do you want to remove this one ?

Comment (ccamel) [reply rust/cosmos_sdk_grpc_client/README.md:6]: Ah ok. I saw `PyPI` and though it was related to python. Need to change the label instead.

Comment (ccamel) [reply rust/cosmos_sdk_grpc_client/README.md:6]: @warstrolo we should be good by now.

Comment (warstrolo) [reply rust/cosmos_sdk_grpc_client/README.md:6]: Yes

**Review by warstrolo — APPROVED** [review]

**Review by ad2ien — APPROVED** [review]

**Review by nperon — APPROVED** [review]

**Review by hleonardtek — APPROVED** [review]

**Pull request merged**